src/driver/sap/SapDriver.ts

```diff
@@ -246,55 +252,37 @@ export class SapDriver implements Driver {
     async connect(): Promise<void> {
         // HANA connection info
         const dbParams = {
-            hostName: this.options.host,
-            port: this.options.port,
-            userName: this.options.username,
-            password: this.options.password,
+            serverNode: `${this.options.host}:${this.options.port}`,
+            uid: this.options.username,
+            pwd: this.options.password,
             ...this.options.extra,
         }
 
-        if (this.options.database) dbParams.databaseName = this.options.database
```

Comment:
Where did this one go?

Also, could we set `dbParams.currentSchema = this.options.schema`?

It's a long time bug that the schema is not set, and this is the fix :)